tests/test_dummy.py

```diff
@@ -21,7 +21,7 @@
 import sys
 import unittest
 
-from builtins import bytes, str
+from builtins import str
```

Comment:
I thought the plan was to rename this to `str_text` everywhere?

Reply:
shouldn't this be fixed then?